common_syms/link.txt

```diff
@@ -3,15 +3,15 @@ gLinkDebugSeed
 gLocalLinkPlayerBlock
 gLinkErrorOccurred
 gLinkDebugFlags
-gFiller_03003074
+gLinkFiller1
```

Comment:
Is it possible to remove some of these fillers and let section alignment handle things?

Reply:
Ah, `gLinkFiller6` and 7 can be yeah. The others would presumably at least require file splits